charts/telepresence/templates/trafficManagerRbac/namespace-scope.yaml

```diff
@@ -1,10 +1,10 @@
+{{- if .Values.managerRbac.create }}
 # This contains the serviceAccount used for the traffic-manager deployment,
-# as well as the various roles + bindings that the traffic-manager needs.
+# as well as the various namespace-scoped roles + bindings that the traffic-manager needs.
 # This will likely expand over time as we move more things from the clients
 # domain into the traffic-manager.  But the good news there is that it will
 # require less permissions in clientRbac.yaml
 
-{{- if .Values.managerRbac.create }}
 apiVersion: v1
```

Comment:
Just a suggestion: 

I know you didn't touch this section, but since this serviceAccount is used in the `namespace-scope.yaml` and the `cluster-scope.yaml`, maybe we should move the serviceAccount into its own yaml? Just so it doesn't seem like this is _only_ used for the objects in namespace-scope.yaml.